### DISCUSSION TITLE
IP ranges of common services?

### DISCUSSION THREAD

**Discussion by chergett** [Security-Onion-Solutions/securityonion#10428] [Unsupported Versions]
I'm curious as to how others are handling the allow-listing of common IP ranges for networks like Google, Microsoft, etc.  I'm trying to think of a way to easily suppress alerts for false positives coming from "trusted" providers.  Perhaps there are services out there that combine this information into an easily digestible format for using with ids rules?  For example, Google lists 214 separate ranges just for the United States.

**Comment by TOoSmOotH** [comment]
I would tune thresholds vs allowing cart blanche access to "trusted" providers. Those trusted providers are used all the time as c2 head ends because they are "trusted". I have seen google drive and twitter used as command and control for many threat groups. They just access a text file shared on google drive or follow a twitter account to know what to do etc.

**Comment by Acewiza** [comment]
Microsoft was hosting [Azure-based Russian election meddlers](https://www.nytimes.com/2020/10/12/us/politics/election-hacking-microsoft.html) in 2020.
